packages/cozy-client/src/queries/dsl.js

```diff
@@ -100,9 +101,9 @@ class QueryDefinition {
    * @returns {QueryDefinition}  The QueryDefinition object.
    */
   sortBy(sort) {
-    if (isString(sort)) {
+    if (!isArray(sort)) {
       throw new Error(
-        'Invalid sort, should be an object (`{ label: "desc"}`), you passed a string.'
+        `Invalid sort, should be an array ([{ label: "desc"}, { name: "asc"}]), you passed ${JSON.stringify(sort)}.`
       )
     }
```

Comment:
Is it a breaking change? Can't we manage both ? Object and array ?

Reply:
No this is not a BC, just a documentation change.

Reply:
sortBy({name: 'asc'}) => Now returns an Error when before no.

Reply:
OK just read the docstring, and we're already expecting an array `   * @param {Array} sort The list of field name and direction pairs.`